controller/sentry/service.go

```diff
@@ -0,0 +1,41 @@
+package sentry
+
+import (
+	"context"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/giantswarm/microerror"
+)
+
+type Config struct {
+	Dsn string
+}
+
+type Service struct {
```

Comment:
Idiomatic would probably be to call this `Sentry` instead of anonymously "Service". I know we did this in the past a lot but it did not turn out to be really nice.

Reply:
done